backend/v1/components/aquariums/routes.py

```diff
@@ -26,47 +26,32 @@ def secret(user: User = Depends(get_admin_user)):
     return "Secret text for admins"
 
 
-router = APIRouter(prefix='/aquariums')
+router = APIRouter(prefix='/Aquariums')
```

Comment:
Use lowercase prefix as per REST API naming convention: https://restfulapi.net/resource-naming/

Reply:
todo in next commit